src/Core/Geometry/IndexedGeometry.hpp

```diff
@@ -63,6 +66,8 @@ struct GeometryIndexLayer : public GeometryIndexLayerBase {
         return false;
     }
 
+    size_t size() override { return _collection.size(); }
```

Comment:
```suggestion
    inline size_t size() override { return _collection.size(); }
```
Use `inline` keyword to avoid dllexport issues, and ease inlining when specialized type is known by the compiler